### PR TITLE
PLAT-228: added sharedbuffers ci

### PR DESCRIPTION
**Pull request by mpucciariello**

### CODE REVIEW

**Comment by mpucciariello** [comment]
run ci

**Comment by mpucciariello** [comment]
run ci

**Comment by mpucciariello** [comment]
run ci

**Comment by mpucciariello** [comment]
run ci

File: jobs/sharedbuffers-ci.groovy [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            steps {
+                script {
+                    docker.image("docker.jampp.com/pythontest-image-builder:${PYTHONTEST_IMAGE_VERSION}").inside(
+                        """\
+                        -v ${WORKSPACE}:/src \
+                        -e REQUIRES_SDIST \
+                        -e PYTHON_PRE_DEPENDENCIES=Cython \
+                        """
+                    ) {
+                        sh "pip install Cython"
```
Comment (marianoSecure) [inline jobs/sharedbuffers-ci.groovy]:
```suggestion
                        sh "pip install Cython==0.29.36"
```

**Review by marianoSecure — APPROVED** [review]

**Pull request merged**